src/app/active/active.component.html

```diff
@@ -0,0 +1 @@
+<app-user-list [usersList]="activeUsers" (changeStatus)="deactivate($event)"></app-user-list>
```

Comment:
`deactivate` should be `deActivate`. Follow camel-case for naming variables / methods